main.rb

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env ruby
+
+require 'pry'
+require './books'
+require './classroom'
+require './corrector'
+require './person'
+require './rental'
+require './students'
+require './teacher'
+
+NEW_LINE = "\n".freeze
+
+$person_list = []
+$book_list = []
+$rentals = []
+
+def menu
+  puts 'Please choose an option by enterin a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given id'
```

Comment:
- [ ] OPTIONAL
Option 6 should read  `List all rentals for a given person id` instead of ` List all rentals for a given id`. Kindly ensure to improve upon this aspect